be/src/exec/pipeline/sink/sink_io_buffer.h

```diff
@@ -105,33 +124,64 @@ class SinkIOBuffer {
     }
 
     static int execute_io_task(void* meta, bthread::TaskIterator<ChunkPtr>& iter) {
+        auto* sink_io_buffer = static_cast<SinkIOBuffer*>(meta);
```

Comment:
I am not sure, the last sinker close the execution queue, but weather it's possible that there are more than one task in queue when closing?

Reply:
Nope. Stop task is the very last task in queue.